examples/feature_engineering/gradient_feature_selector/benchmark_test.py

```diff
@@ -34,7 +38,7 @@
 
 class Benchmark():
 
-    def __init__(self, files, test_size = 0.2):
+    def __init__(self, files=None, test_size = 0.2):
```

Comment:
inconsistent spaces on parameters,  better to remove the spaces after `test_size`

Reply:
ok